flamedisx/source.py

```diff
@@ -276,6 +297,8 @@ def __init__(self,
         if not _skip_tf_init:
             self.trace_differential_rate()
 
+        self.source_copy = deepcopy(self)
```

Comment:
I think we don't need this yet, let's have a look at this in your next PR -- maybe there is another way

Reply:
Removed for now. I'd advocate that we please don't squash-merge this: I'd like a copy of my thought process to be preserved, if this comes back to bite in future